scripts/pipelines/gcloud/templates/gke/gke-provisioning.yml.template

```
@@ -0,0 +1,41 @@
+steps:
+- name: gcr.io/cloud-builders/gsutil
+  entrypoint: bash
+  args:
+  - -c
+  - |
+    #Install terraform
+    apt-get update
+    apt-get install -y gnupg software-properties-common wget dnsutils
+    wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor | tee /usr/share/keyrings/hashicorp-archive-keyring.gpg
+    gpg --no-default-keyring --keyring /usr/share/keyrings/hashicorp-archive-keyring.gpg --fingerprint
+    echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/hashicorp.list
+    apt-get update
+    apt-get install terraform
+
+    #Terraform init
+    cd .terraform/gke/
+    terraform init -input=false
+
+    if [ "${_OPERATION}" == "create" ]; then
+      #Create cluster
+      terraform apply -input=false -auto-approve
+      gcloud components install kubectl
+      gcloud container clusters get-credentials $(terraform output -raw kubernetes_cluster_name) --region $(terraform output -raw region) --project=${PROJECT_ID}
+      ../../.pipelines/scripts/setup-helm.sh
+      ../../.pipelines/scripts/install-nginx-ingress.sh
```

```suggestion
      .pipelines/scripts/setup-helm.sh
      .pipelines/scripts/install-nginx-ingress.sh
```
With terraform in the PATH, we simplify the paths to common scripts

---

Terraform finally not in PATH, so this remains as it is